_studio/mfx_lib/shared/include/mfx_mpeg2_encode_interface.h

```diff
@@ -149,7 +149,7 @@ namespace MfxHwMpeg2Encode
     public:
         virtual ~DriverEncoder(){}
 
-        virtual mfxStatus QueryEncodeCaps(ENCODE_CAPS & caps, mfxU8 codecProfileType) = 0;
+        virtual mfxStatus QueryEncodeCaps(ENCODE_CAPS & caps) = 0;
```

Comment:
can we drop return status here? From implementation it seems that ```void``` is more suitable